xterm/config

```diff
@@ -1 +1,2 @@
 base_port=555
+noflavors=0
```

Comment:
This should be off by default, at least initially. We don't know if the user's shell even supports aliases or the PS1 var, and also I want to get a basic working version of this module out to users before we add a lot of fancy stuff.

Reply:
`bash` does support all of this. If not, it will be just ignored. But alright, I reverted that one. Let's keep it off by default for now for regular users.